Assets/Scripts/Sample.cs

```diff
@@ -19,6 +18,12 @@ private void Start()
         {
             Debug.Log(items[i]);
         }
+
+        var positions = new List<Vector3>(20);
+        var number = Random.Range(0, 10);
+        for (int i = 0; i < 20; i++)
+        {
+        }
```

Comment:
Missing:
1. Generating random vectors
2. Adding elements to the list.
3. Vector addition
4. Removal of vectors.